### PR TITLE
Added Extra Parameters to Facebook Shares

### PR DESCRIPTION
**Pull request by Zaldor**
Added: photo, name, and description
but descriptions shows unwanted html tags
P.S.
First time ever contributing, not a programmer.
sorry, I hope it helps

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Comment by Progi1984** [comment]
Hello @Zaldor , thanks for your contribution, would you mind to rebase your pull request? Thanks

**Review by PierreRambaud — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

File: ps_sharebuttons.php [hunk 1]
```diff
@@ -176,7 +177,7 @@ public function getWidgetVariables($hookName, array $params)
             $social_share_links['facebook'] = [
                 'label' => $this->trans('Share', [], 'Modules.Sharebuttons.Shop'),
                 'class' => 'facebook',
-                'url' => 'https://www.facebook.com/sharer.php?u='.$sharing_url,
+                'url' => 'http://www.facebook.com/sharer.php?u='.$sharing_url.'&picture='.$sharing_img.'&title='.$sharing_name.'&description='.$sharing_desc,
```
Comment (atomiix) [inline ps_sharebuttons.php:180]: I think we can keep with `https` here.
```suggestion
                'url' => 'https://www.facebook.com/sharer.php?u='.$sharing_url.'&picture='.$sharing_img.'&title='.$sharing_name.'&description='.$sharing_desc,
```

**Comment by Progi1984** [comment]
Ping @Zaldor

**Comment by matks** [comment]
Hi,

Since we had no news from you for more than 5 months, I will follow [our guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/#stale-pull-requests) and I'll close this pull request as it seems stale. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**